### PR TITLE
mopidy-podcast: init at 3.0.0

### PR DESCRIPTION
**Pull request by ghost**
Mopidy extension for browsing and playing podcasts

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/audio/mopidy/podcast.nix
Comment (mweinelt) [inline pkgs/applications/audio/mopidy/podcast.nix:22]:
```suggestion
    homepage = "https://github.com/tkem/mopidy-podcast";
```

This is the actual homepage. If you visit their pypi page it links there.

File: pkgs/applications/audio/mopidy/podcast.nix
Comment (mweinelt) [inline pkgs/applications/audio/mopidy/podcast.nix:19]:
```suggestion
  checkInputs = with python3Packages; [
    pytestCheckHook
  ];
```

We really want tests enabled for python modules and application, to have some additional guarantees that they work.

**Review by mweinelt — CHANGES_REQUESTED** [review]
We require `meta.maintainers` to be set in newly submitted packages. Your GitHub account looks like a throw-away, is that correct? If so I'm not sure if you'd be willing to maintain this package. Please clarify.

File: pkgs/applications/audio/mopidy/podcast.nix
Comment (daneads) [reply pkgs/applications/audio/mopidy/podcast.nix]: Thanks much for your suggestions! I've updated the maintainer + used a real name account, and added the changes requested above.

**Review by mweinelt — COMMENTED** [review]
Looks pretty good already. Can you fix up two more things?

The maintainer-list change should be its own commit, since it relates to another file in the repository. And the tests change can be squashed into the package commit itself.

**Comment by daneads** [comment]
> Looks pretty good already. Can you fix up two more things?
> 
> The maintainer-list change should be its own commit, since it relates to another file in the repository. And the tests change can be squashed into the package commit itself.

@mweinelt Ok to keep the maintainer-list in this PR? I made a separate PR but was told it can be part of this one.

> You can just include this commit in the PR in which you add youself to some meta.maintainers list.

https://github.com/NixOS/nixpkgs/pull/121503#issuecomment-830803859

**Comment by mweinelt** [comment]
Both ways are fine, but the maintainer-list change needs to be a separate commit. I can merge #121503 and you can then rebase on top of that, if you'd prefer that.

**Comment by daneads** [comment]
Sounds great. Once that's merged I'll rebase this.

**Review by mweinelt — APPROVED** [review]

**Pull request merged**
